components/brave_wallet_ui/common/constants/mocks.ts

```diff
@@ -434,18 +460,31 @@ export const mockSolDappSignAllTransactionsRequest: BraveWallet.SignAllTransacti
           'accountMetas': [
             {
               'pubkey': mockSolanaAccount.address,
+              'addrTableLookupIndex': undefined,
               'isSigner': true,
               'isWritable': true
             },
             {
               'pubkey': mockSolanaAccount.address,
+              'addrTableLookupIndex': undefined,
               'isSigner': true,
               'isWritable': true
             }
           ],
           'data': [2, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 0],
           'decodedData': undefined
         }],
+        'version': BraveWallet.SolanaMessageVersion.kLegacy,
+        'messageHeader': {
+          'numRequiredSignatures': 1,
+          'numReadonlySignedAccounts': 0,
+          'numReadonlyUnsignedAccounts': 1
+        },
+        'staticAccountKeys': [
+          mockSolanaAccount.address,
+          '11111111111111111111111111111111'
+        ],
+        'addressTableLookups': [],
```

Comment:
Should we add a v0 tx mock so we can test aginst it?

Reply:
`unit-test` added in https://github.com/brave/brave-core/pull/17542/commits/e3febc26a3c28f3edc2bb2ae387d3ca8f731ac7c